beacon-chain/sync/pending_attestations_queue.go

```diff
@@ -28,14 +28,14 @@ import (
 var processPendingAttsPeriod = slots.DivideSlotBy(2 /* twice per slot */)
 var pendingAttsLimit = 10000
 
-// This processes pending attestation queues on every `processPendingAttsPeriod`.
-func (s *Service) processPendingAttsQueue() {
+// This processes pending attestation queues on every processPendingAttsPeriod.
+func (s *Service) runPendingAttsQueue() {
```

Comment:
I think `run` is a better name given that attestations are processed in a continuous manner